docs/using_helm.md

```diff
@@ -95,31 +95,75 @@ simplest, it takes only one argument: The name of the chart.
 $ helm install stable/mariadb
 Fetched stable/mariadb-0.3.0 to /Users/mattbutcher/Code/Go/src/k8s.io/helm/mariadb-0.3.0.tgz
 NAME: happy-panda
-LAST DEPLOYED: Wed Sep 28 12:32:28 2016
+LAST DEPLOYED: Fri Oct 19 12:40:03 2018
 NAMESPACE: default
 STATUS: DEPLOYED
 
-Resources:
-==> extensions/Deployment
-NAME                     DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-happy-panda-mariadb   1         0         0            0           1s
+RESOURCES:
+==> v1/Pod(related)
+NAME                                READY  STATUS   RESTARTS  AGE
+happy-panda-mariadb-master-0  0/1    Pending  0         0s
+happy-panda-mariadb-slave-0   0/1    Pending  0         0s
 
 ==> v1/Secret
-NAME                     TYPE      DATA      AGE
-happy-panda-mariadb   Opaque    2         1s
+
+NAME                       AGE
+happy-panda-mariadb  0s
+
+==> v1/ConfigMap
+happy-panda-mariadb-master  0s
```

Comment:
There's a current regression in v2.11.0 for the status output (see #4791). I think we should hold off until we release v2.11.1 with this fix before updating the docs on `helm status`'s output, or use v2.10.0 instead.

Any thoughts/opinions on this?

Reply:
As I use the latest version of the helm. I do not know is it ok to keep inconsistent output in example.
Any idea ?